test/parallel/test-whatwg-url-searchparams-getall.js

```diff
@@ -25,15 +25,17 @@ test(function() {
 
 test(function() {
     var params = new URLSearchParams('a=1&a=2&a=3&a');
-    assert_true(params.has('a'), 'Search params object has name "a"');
+    assert_true(params.has('a'), `Search params object doesn't have name "a"`);
```

Comment:
This will not pass `make lint` given that it is a plain string without any replacements. Please simply avoid the use of the contraction and make it a normal string.. e.g. `'Search params object does not have name "a"'`.